### PR TITLE
GSLUX-659: Display attributions

### PR DESCRIPTION
**Pull request by AlitaBernachot**
### JIRA issue

https://jira.camptocamp.com/browse/GSLUX-659

### Description

- [x] Enable attributions on map

### CODE REVIEW

File: src/assets/main.css [hunk 1]
```diff
@@ -281,6 +285,14 @@
   .lux-time-datepicker {
     @apply border-[#767676] border-[1px] pl-1;
   }
+
+  .lux-attribution {
```
Comment (tkohr) [inline src/assets/main.css:289]: If `lux-attribution` is only used once, can't we achieve the same with tailwind classes within the component?

**Review by tkohr — APPROVED** [review]
Thanks, @AlitaBernachot! Works as expected, just left a comment regarding CSS organization. An e2e test could also be nice, but not mandatory. I'll let you decide.

File: src/assets/main.css (hunk 1 above)
Comment (AlitaBernachot) [reply src/assets/main.css]: I prefer having all my styles in a css file, but ok, moved to component template if you think it is better like this 🙂

**Comment by AlitaBernachot** [comment]
> Thanks, @AlitaBernachot! Works as expected, just left a comment regarding CSS organization. An e2e test could also be nice, but not mandatory. I'll let you decide.

Thx for the review! I have added some e2e testing.

File: src/assets/main.css (hunk 1 above)
Comment (tkohr) [reply src/assets/main.css]: Thanks, I'm just worried, we'll and up with a very long CSS file like in the legacy, making styles less readable.

**Pull request merged**